bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/HomeConnectWasherHandler.java

```diff
@@ -96,13 +114,26 @@ protected void configureEventHandlers(Map<String, EventHandler> handlers) {
                 event -> getThingChannel(CHANNEL_WASHER_SPIN_SPEED).ifPresent(channel -> updateState(channel.getUID(),
                         event.getValue() == null ? UnDefType.UNDEF : new StringType(event.getValue()))));
         handlers.put(EVENT_WASHER_IDOS_1_DOSING_LEVEL,
-                event -> getThingChannel(CHANNEL_WASHER_IDOS1).ifPresent(channel -> updateState(channel.getUID(),
+                event -> getThingChannel(CHANNEL_WASHER_IDOS1_LEVEL).ifPresent(channel -> updateState(channel.getUID(),
                         event.getValue() == null ? UnDefType.UNDEF : new StringType(event.getValue()))));
         handlers.put(EVENT_WASHER_IDOS_2_DOSING_LEVEL,
-                event -> getThingChannel(CHANNEL_WASHER_IDOS2).ifPresent(channel -> updateState(channel.getUID(),
+                event -> getThingChannel(CHANNEL_WASHER_IDOS2_LEVEL).ifPresent(channel -> updateState(channel.getUID(),
                         event.getValue() == null ? UnDefType.UNDEF : new StringType(event.getValue()))));
     }
 
+    @Override
+    protected boolean isLinkedChannelsForUnsupportedFeatures() {
```

Comment:
I'm not sure about the method name. Unfortunately, I can't think of a better name.

Reply:
Maybe `isChannelLinkedToProgramOptionNotFullySupportedByApi` ?

Reply:
It's long but better. Could you please use it.

Reply:
Done